lib/Auth/Process/AuthnAuthority.php

```diff
@@ -50,29 +55,29 @@ public function __construct($config, $reserved)
 
         if (array_key_exists('attribute', $config)) {
             if (!is_string($config['attribute'])) {
-                SimpleSAML_Logger::error(
+                Logger::error(
                     "[authnauthority] Configuration error: 'attribute' not a string literal");
-                throw new SimpleSAML_Error_Exception(
+                throw new \Exception(
```

Comment:
Remove leading backslash. See this: https://symfonycasts.com/screencast/php-namespaces-in-120-seconds/namespaces.

Reply:
Used `new Error\Exception(`